HandheldCompanion/Controls/GameView.xaml

```diff
@@ -0,0 +1,293 @@
+﻿<UserControl
```

Comment:
Not sure whether this should be a UserControl or DataTemplate. I think it's something that will have to be balanced.